iac-extensions/terraform/src/test/resources/checks/Azure/ResourceSpecificAdminAccountCheck/test.tf

```diff
@@ -0,0 +1,54 @@
+resource "azurerm_container_registry" "sensitive" {
+  # Noncompliant@+1 {{Make sure that enabling an administrative account or administrative permissions is safe here.}}
+  admin_enabled  = true
+# ^^^^^^^^^^^^^^^^^^^^^
+}
+
+resource "azurerm_container_registry" "disabled" {
+  admin_enabled  = false
+}
+
+resource "azurerm_container_registry" "missing" {
+}
+
+resource "azurerm_batch_pool" "sensitive_pool" {
+  name = "sensitive"
+  start_task {
+    user_identity {
+      auto_user {
+        elevation_level = "Admin" # Noncompliant
+      }
+    }
+  }
+}
+
+resource "azurerm_batch_pool" "compliant_pool" {
+  name = "sensitive"
+  start_task {
+    user_identity {
+      auto_user {
+        elevation_level = "NonAdmin"
+      }
+    }
+  }
+}
+
+resource "azurerm_batch_pool" "compliant_pool" {
+  name = "sensitive"
+  start_task {
+    user_identity {
+    }
+  }
+}
+
+resource "azurerm_batch_pool" "compliant_pool" {
+  name = "sensitive"
+  start_task {
+  }
+}
+
+resource "azurerm_batch_pool" "compliant_pool" {
```

Comment:
If there is a nested property, say:
**block1.block2.block.property**
Is it customary to test for all the add all partial paths along the way (like in the last 3 TCs)?